src/hooks/useActionCenterSubscribers/useArbitrumBridgeTransactionHistorySubscriber.tsx

```diff
@@ -0,0 +1,157 @@
+import type { ChildToParentMessageReader, ChildToParentTransactionEvent } from '@arbitrum/sdk'
+import { ChildToParentMessageStatus, ChildTransactionReceipt } from '@arbitrum/sdk'
+import { arbitrumChainId, ethChainId, toAccountId } from '@shapeshiftoss/caip'
+import { getEthersV5Provider } from '@shapeshiftoss/contracts'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import type { Query } from '@tanstack/react-query'
+import { useQueries } from '@tanstack/react-query'
+import { uuidv4 } from '@walletconnect/utils'
+import { useCallback, useEffect } from 'react'
+
+import type { TxMetadata } from '../../../packages/unchained-client/src/evm/parser/arbitrumBridge'
+
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import { ActionStatus, ActionType } from '@/state/slices/actionSlice/types'
+import { selectArbitrumWithdrawTxs } from '@/state/slices/selectors'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+const AVERAGE_BLOCK_TIME_BLOCKS = 1000
+
+type ClaimStatusResult = {
+  event: ChildToParentTransactionEvent
+  message: ChildToParentMessageReader
+  status: ChildToParentMessageStatus
+  timeRemainingSeconds: number | undefined
+}
+
+// Helper hook to detect ArbitrumBridge withdrawals from transaction history and create initial actions
+// TODO: Remove this in next PR when claim tab is removed - this replaces "poor man's migration"
```

Comment:
Not super concerned about cleanliness of this guy - here for testing purposes - either way, mostly a 1/1 copypastaroo of existing logic!